api/src/hyp3_api/handlers.py

```diff
@@ -34,6 +39,13 @@ def post_jobs(body, user):
     except QuotaError as e:
         return problem(400, 'Bad Request', str(e))
 
+    try:
+        check_granules_exists([job['job_parameters']['granule'] for job in body['jobs']])
```

Comment:
could include ` if job['job_type'] == 'RTC_GAMMA'`, not strictly necessary yet though

Reply:
and maybe break up the creation of the list and the function call:
```
granules = ....
check_granules_exist(granules)
```